README.md

```diff
@@ -1,37 +1,91 @@
-1. providing feedback for the HMRC Customer Contact team ('Survey')
-1. reporting accessibility problems with services ('Report an accessibility problem')
-
-`contact-frontend` is responsible for hosting forms related to the purposes listed above, validating the input, and
-passing user requests to downstream services - to Deskpro for 'Is this page not working properly?', 'Help and contact', 
-'Send your feedback' and 'Report an accessibility problem', and an internal auditing service for 'Survey' (contact
- [#team-platui](https://hmrcdigital.slack.com/messages/team-plat-ui/) if you require more information about this).
-
-# Contents
-   * [Roadmap](#roadmap)
-   * [Forms provided by the Customer Contact Subsystem](#forms-provided-by-the-customer-contact-subsystem)
-      * [Contacting HMRC - <em>Is this page not working properly?</em>](#contacting-hmrc---get-help-with-this-page)
-      * [Contacting HMRC - <em>Help and contact</em>](#contacting-hmrc---help-and-contact)
-      * [Providing feedback about Digital Customer Support Team](#providing-feedback-about-digital-customer-support-team)
-      * [Providing Beta feedback about services](#providing-beta-feedback-about-services)
-      * [Report an accessibility problem](#report-an-accessibility-problem) 
-   * [Integration guide](#integration-guide)
-      * [referer header forwarding to Deskpro](#referer-header-forwarding-to-deskpro)
-      * [Creating own customer contact forms](#creating-own-customer-contact-forms)
-   * [Other relevant details](#other-relevant-details)
-      * [User details attached to the ticket](#user-details-attached-to-the-ticket)
-      * [Related projects, useful links](#appendix__linx)
-      * [Slack](#appendix__links__slack)
-
-# Forms provided by the Customer Contact Subsystem <a name="forms-provided-by-the-customer-contact-subsystem"></a>
-
-## Contacting HMRC - *Is this page not working properly?* <a name="contacting-hmrc---get-help-with-this-page"></a>
+# contact-frontend
+
+## Overview
+This service provides forms that enable users to contact HMRC for five distinct reasons:
+1. reporting a technical problem ('Is this page not working properly?')
+1. providing feedback on a beta service ('Send your feedback')
+1. getting help with their tax account ('Help and contact')
+1. providing post-support feedback to the HMRC Digital Technical Support Team ('Survey')
+1. reporting an accessibility problem with a service ('Report an accessibility problem')
+
+`contact-frontend` will validate the user's input (for example, to ensure that email addresses are valid),
+and pass user requests to the relevant downstream service - to Deskpro for 'Is this page not working properly?',
+'Help and contact', 'Send your feedback' and 'Report an accessibility problem',
+and to an internal auditing service for 'Survey' (contact
+[#team-platui](https://hmrcdigital.slack.com/messages/team-plat-ui/)
+if you require more information about this).
+
+## Contents
```

Comment:
removed dead links, created integration guide section for service teams, pushed example forms (including ones they don't need to know about) down to Appendix